test/conf/creds.conf.sample

```diff
@@ -1,12 +0,0 @@
-host = "dev-test-1"
```

Comment:
Pls keep this file. It is just sample for other to know the format

Reply:
I have kept this file for the new commit change